windows/icu-easyrpg/portfile.cmake

```diff
@@ -29,7 +29,7 @@ vcpkg_download_distfile(
     ARCHIVE
     URLS "https://ci.easyrpg.org/job/icudata/lastSuccessfulBuild/artifact/icudata.tar.gz"
```

Comment:
just thinking:
Maybe we should use the package `icudata-${ICU_VERSION_MAJOR}.tar.gz` on https://easyrpg.org/downloads/icudata/

This way the hash does not change and we only need to take care when updating the vcpkg version.

Reply:
sounds like a good idea.